packages/proxy/lib/http/index.ts

```diff
@@ -107,10 +107,10 @@ export type HttpMiddlewareThis<T> = HttpMiddlewareCtx<T> & ServerCtx & Readonly<
   skipMiddleware: (name: string) => void
 }>
 
-export function _runStage (type: HttpStages, ctx: any, onError) {
+export function _runStage (type: HttpStages, ctx: any, onError: Function, res?: CypressOutgoingResponse) {
```

Comment:
`res` is already available as `ctx.res`, no need to pass it here

Reply:
handy!